src/components/Apps/AppItem.jsx

```diff
@@ -30,7 +30,14 @@ export class AppItem extends React.Component {
   }
 
   buildAppUrl(href) {
-    const url = new URL(href)
+    let url
+    try {
+      url = new URL(href)
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error.message)
+      return ''
```

Comment:
`return null` to me is clearer. It will also prevent the AppLinker to add an empty href to the underlying element. Only a null prop which will not be trasnformed into an attribute.

Reply:
Ok. So we need more test to see how it behaves with an URL being null. 👍

Reply:
Done ! :tada: